controllers/articles.controller.js

```diff
@@ -15,8 +15,8 @@ exports.getArticlesByIDCon = (req, res, next) => {
 }
 
 exports.getArticlesCon = (req, res, next) => {
-    const {sort_by, order} = req.query
-    getArticlesMod(sort_by, order).then((articles) => {
+    const {sort_by, order, topic} = req.query
+    getArticlesMod(sort_by, order, topic).then((articles) => {
```

Comment:
To pass the suggested test,  there should be a way of checking if the topic exists. 
The existence of a topic must come from the db, not to be hard coded.